CHANGES/287.bugfix

```diff
@@ -0,0 +1 @@
+Add the fake not namespaced pulp_cli to the distribution on pypi to make it able to be consumed by setuptools<40
```

Comment:
"Always add" or "Unconditionally add" is probably more precise :)

Reply:
Well it adds it to what will be on pypi later...